tests/contrib/sensors/test_redis_pub_sub_sensor.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+
+import unittest
+from airflow import DAG
+from airflow import configuration
+from airflow.contrib.sensors.redis_pub_sub_sensor import RedisPubSubSensor
+from airflow.utils import timezone
+from mock import patch
+from mock import call
+from mock import MagicMock
```

Comment:
should we use single line
```suggestion
from mock import MagicMock patch call
```

Reply:
Done.